Analysis/Tasks/eventSelection.cxx

```diff
@@ -123,12 +133,97 @@ struct EventSelectionTask {
     }
 
     // Fill event selection columns
-    evsel(alias, bbT0A, bbT0C, bbV0A, bbV0C, bgV0A, bgV0C, bbZNA, bbZNC, bbFDA, bbFDC, bgFDA, bgFDC);
+    int64_t foundFT0 = -1; // this column is not used in run2 analysis
+    evsel(alias, bbT0A, bbT0C, bbV0A, bbV0C, bgV0A, bgV0C, bbZNA, bbZNC, bbFDA, bbFDC, bgFDA, bgFDC, foundFT0);
   }
 };
 
-WorkflowSpec defineDataProcessing(ConfigContext const&)
+// helper table and task for BCs<-->FT0s matching
+namespace o2::aod
 {
-  return WorkflowSpec{
-    adaptAnalysisTask<EventSelectionTask>("event-selection")};
+DECLARE_SOA_INDEX_TABLE_USER(SparseBCsFT0s, BCs, "MA_SP_BCS_FT0S", o2::aod::indices::BCId, o2::aod::indices::FT0Id);
 }
+
+struct EventSelectionTaskRun3Helper {
+  Builds<aod::SparseBCsFT0s> matchedBCsFT0s;
```

Comment:
@aalkin The building is automatic now, isn't it? So it this task needed?

Reply:
Similar to what we have with extended tables, automatic building needs to be added for a given type in `AODReaderHelpers.cxx`

Reply:
Thanks. OK, then we keep it here until we have the final solution of the matching.